tests/Aspire.Hosting.Azure.Tests/AzureBicepResourceTests.cs

```diff
@@ -1412,22 +1390,12 @@ public async Task AsAzurePostgresFlexibleServerViaPublishMode()
         var pwd = builder.AddParameter("pwd", secret: true);
 
 #pragma warning disable CS0618 // Type or member is obsolete
-        IResourceBuilder<AzurePostgresResource>? azurePostgres = null;
-        var postgres = builder.AddPostgres("postgres", usr, pwd).AsAzurePostgresFlexibleServer((resource, _, _) =>
-        {
-            Assert.NotNull(resource);
-            azurePostgres = resource;
-        });
+        var postgres = builder.AddPostgres("postgres", usr, pwd).AsAzurePostgresFlexibleServer();
         postgres.AddDatabase("db", "dbName");
 #pragma warning restore CS0618 // Type or member is obsolete
 
         var manifest = await ManifestUtils.GetManifestWithBicep(postgres.Resource);
 
-        // Setup to verify that connection strings is acquired via resource connectionstring redirct.
```

Comment:
Should do the same thing here as the AsAzureSqlDatabase tests - get the azure resource from the annotation, and set its outputs.